### PR TITLE
feat: add Sandra Toftemo to list of employees

### PR DESCRIPTION
**Pull request by Xillians**

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```
Comment (simenandre) [inline pnpm-lock.yaml:1]: Did you run this with a newer version of `pnpm` than v8.5.1? This should probably not be in this PR, but hey, if it's needed – let's get it in there! :)

File: src/routes/(default)/about/employees.svelte [hunk 2]
```diff
@@ -74,6 +75,25 @@
...
+	givenName="Sandra Viktoria"
+	familyName="Toftemo"
+	position="Programvareutvikler"
+	phone="+47 406 30 694"
+	email="sandra.toftemo@bjerk.io"
+	image={sandra}
+	socialMedia={[
+		{
+			type: 'linkedin',
+			url: 'https://www.linkedin.com/in/sandra-viktoria-doseth-toftemo-b9b946227/'
```
Comment (simenandre) [inline src/routes/(default)/about/employees.svelte:88]: Tip: There's a [way to set a personal URL on LinkedIn](https://www.linkedin.com/help/linkedin/answer/a542685/manage-your-public-profile-url?lang=en)

**Review by simenandre — APPROVED** [review]

**Review by HanBrustad — APPROVED** [review]

**Review by braaar — APPROVED** [review]
SBU!

(Ser bra ut)

**Review by AnneMatilde — COMMENTED** [review]
Første PR! :tada:

**Pull request merged**